packages/cyberstorm/src/components/Button.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { MouseEventHandler } from "react";
+import styles from "./componentStyles/Button.module.css";
+
+export interface ButtonProps {
+  label?: string;
+  leftIcon?: string;
+  rightIcon?: string;
+  buttonStyle?: string;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+/**
+ * Cyberstorm Button component
+ */
+export const Button: React.FC<ButtonProps> = (props) => {
+  const { label, leftIcon, rightIcon, buttonStyle, onClick } = props;
+  const stylesImport = styles;
+
+  return (
+    <button className={"button button__" + buttonStyle} onClick={onClick}>
+      {leftIcon}
```

Comment:
Might be a good idea to pass some default button related classes for the icon elements.

Reply:
Why is this? 🤔

Reply:
To make it easier to hook on it if needed. Something I've personally found better to overdo than not.

Makes sense also to have every child have classes than just some - and since we're creating a universal library for ourselves, would make sense to me to have the components follow the same principles so you don't have to guess or have to do the same thing differently depending on a component.

This all comes down to fact that imo we should agree on some guidelines so you don't have to think about these kind of things after they've been set. Thoughts?

Reply:
Now passing from the story